plugins/module_utils/vmware_nsxt.py

```diff
@@ -44,62 +45,76 @@ def request(url, data=None, headers=None, method='GET', use_proxy=True,
     else:
         client_cert = None
 
-    ca_path = get_certificate_file_path('NSX_MANAGER_CA_PATH')
+    if method == 'GET':
+        return get_all_results(
+            url, data, headers, method, use_proxy, force, last_mod_time,
+            timeout, validate_certs, url_username, url_password, http_agent,
+            force_basic_auth, ignore_errors, client_cert)
+    return _request(
+        url, data, headers, method, use_proxy, force, last_mod_time, timeout,
+        validate_certs, url_username, url_password, http_agent,
+        force_basic_auth, ignore_errors, client_cert)
+
+def get_all_results(
+        url, data, headers, method, use_proxy, force, last_mod_time, timeout,
+        validate_certs, url_username, url_password, http_agent,
+        force_basic_auth, ignore_errors, client_cert):
     NULL_CURSOR_PREFIX = '0000'
+    rc, resp = _request(
+        url, data, headers, method, use_proxy, force, last_mod_time, timeout,
+        validate_certs, url_username, url_password, http_agent,
+        force_basic_auth, ignore_errors, client_cert)
+    if rc != 200:
+        return rc, None
+    cursor = resp.get('cursor', NULL_CURSOR_PREFIX)
```

Comment:
Do you think using `NULL_CURSOR_PREFIX` is really necessary?

Reply:
I think we can remove it